collectionchallenge.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=pokemon', 'root', 'password' );
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats`;");
```

Comment:
maybe make a more specific variable name for your query as I'm guessing you'll want to just pull items by name and have php pull the rest which will require multiple queries or different fetches.

Reply:
Currently pulling all of my data with one query

Reply:
Updated anyway